src/main/java/uk/ac/ebi/eva/pipeline/configuration/jobs/flows/AnnotationFlowConfiguration.java

```diff
@@ -44,32 +43,22 @@
  */
 @Configuration
 @EnableBatchProcessing
-@Import({GenerateVepAnnotationStepConfiguration.class, LoadVepAnnotationStepConfiguration.class, AnnotationMetadataStepConfiguration.class})
+@Import({GenerateVepAnnotationStepConfiguration.class, AnnotationMetadataStepConfiguration.class})
 public class AnnotationFlowConfiguration {
 
     @Autowired
     @Qualifier(GENERATE_VEP_ANNOTATION_STEP)
     private Step generateVepAnnotationStep;
 
     @Autowired
-    @Qualifier(LOAD_VEP_ANNOTATION_STEP)
-    private Step annotationLoadStep;
-
-    @Autowired
     @Qualifier(LOAD_ANNOTATION_METADATA_STEP)
     private Step annotationMetadataStep;
 
     @Bean(VEP_ANNOTATION_FLOW)
     public Flow vepAnnotationFlow() {
-        EmptyVepOutputDecider emptyVepOutputDecider = new EmptyVepOutputDecider();
-
         return new FlowBuilder<Flow>(VEP_ANNOTATION_FLOW)
                 .start(generateVepAnnotationStep)
```

Comment:
yes, the reader will return null on the first batch, and the processor and writer won't be called